contents/forms/checkbox/metadata.json

```
@@ -0,0 +1,10 @@
+{
+  "en": {
+    "title": "Checkbox",
+    "description": "Component used to allow the user to select multiple values from multiple options."
```

It would be better if the title is in plural form.
```suggestion
    "title": "Checkboxes",
    "description": "Component used to allow the user to select multiple values from multiple options."
```

---

I fixed.